src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/math/Fraction.java

```diff
@@ -21,28 +24,25 @@
 
 	// should be only called if denom is positive and num & denom are coprime
```

Comment:
```suggestion
	/**
	 * Should be only called if denominator is positive and numerator & denominator are coprime.
	 */
```

Promoted to a full comment